docs/control_center_docs/control_center_start_guide.rst

```diff
@@ -0,0 +1,737 @@
+see :ref:`manual_velocity_control_mechanisms`
+
+Voltage
+=============
+This commands the module to a specific drive voltage. A positive voltage always goes in the counterclockwise direction, and a negative voltage always goes clockwise.
+
+ESC Input
+=============
+This commands the module to spin in the same way that a typical hobby protocol used by a flight controller would, i.e. it sends an IQUART throttle command. 
+So, setting this to 0.5 will send the module a 50% throttle command with the configuration specified in this example. However, because this 
+test is meant to simulate commands from a flight controller, it is affected by the Advanced Arming feature. That means that setting this 
+parameter will not cause the module to spin until the module has armed. **By default Vertiq speed modules require 10 consecutive throttle 
+commands between 0% and 7.5% to arm. So, to arm your module using the ESC Input parameter, set the ESC Input to 0.05 to send a 5% throttle 
+command, and click the set arrow 10 times**. On the 10th time you click the set arrow, the module should play its 2 note arming song, and begin spinning. 
+Now that the module is armed, any new throttle commands sent using the ESC Input parameter will change how it is spinning. For example, setting ESC 
+Input to 0.5 should set the module to the same speed as setting the Voltage parameter to 2.5V. This is because the module is in Voltage mode with a 
+Max Volts of 5V, so a 50% throttle command will set the module to spin with a drive voltage of 2.5V. For more information on how to configure 
+the module to properly interpret throttle commands, see the Throttle Commands and Modes section of the Feature Reference Manual. For more information on 
+arming and disarming the module, refer to the Advanced Arming section of the Feature Reference Manual.
+
+Try it Yourself
+==================
+Try out some of these parameters and observe how the module spins. For example, try setting the Voltage to 2.5V, then set the ESC Input to 0.5. 
+You should observe the module staying at the same speed. You can repeat this with 5.0V and a 1.0 ESC input. Remember, to accept commands from ESC 
+Input, the module must be armed as described above. The module is now set up for basic testing with the Control Center. For integrating with a flight 
+controller, more setup is needed. Refer to the PWM and DSHOT Control with a Flight Controller and the DroneCAN Integration with a PX4 Flight Controller 
+for more details on integrating with flight controllers.
+
+*******************************************
+Updating Firmware with the Control Center
+*******************************************
+Flashing new firmware onto your module is done through the FIRMWARE tab in the Control Center, available on the left side.
+
+All module firmware can be found on your product's page on `Vertiq's website <https://www.vertiq.co/>`_.
+Screen Capture Example
```

Comment:
I think Screen Capture Example was meant to be a heading, but it's just regular text so it kind of comes out of nowhere right now